playbooks/configure_secrets_backend.yml

```diff
@@ -0,0 +1,95 @@
+  gather_facts: false
+  tasks:
+    - name: Load defaults
+      ansible.builtin.include_role:
+        name: pattern_defaults
+
+    - name: Print start message
+      ansible.builtin.shell:
+        cmd: |
+          echo "Setting secrets backend to '{{ secrets_backing_store }}'..." > /dev/tty
+
+    - name: Set backend in values-global.yaml
+      ansible.builtin.include_role:
+        name: set_secret_backend
+
+    - name: Configure resources for vault backend
+      when: secrets_backing_store == 'vault'
+      block:
+        - name: Ensure vault app present
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: vault
+            application_state: present
+
+        - name: Ensure golang-external-secrets app present
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: golang-external-secrets
+            application_state: present
+
+        - name: Ensure validated-patterns-secrets namespace absent
+          ansible.builtin.include_role:
+            name: manage_secret_namespace
+          vars:
+            app_namespace: validated-patterns-secrets
+            namespace_state: absent
+
+    - name: Configure resources for kubernetes backend
+      when: secrets_backing_store == 'kubernetes'
+      block:
+        - name: Ensure validated-patterns-secrets namespace present
+          ansible.builtin.include_role:
+            name: manage_secret_namespace
+          vars:
+            app_namespace: validated-patterns-secrets
+            namespace_state: present
+
+        - name: Ensure vault app absent
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: vault
+            application_state: absent
+
+        - name: Ensure golang-external-secrets app present
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: golang-external-secrets
+            application_state: present
+
+    - name: Configure resources for no backend
+      when: secrets_backing_store == 'none'
+      block:
+        - name: Ensure vault app absent
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: vault
+            application_state: absent
+
+        - name: Ensure golang-external-secrets app absent
+          ansible.builtin.include_role:
+            name: manage_secret_app
+          vars:
+            application: golang-external-secrets
+            application_state: absent
+
+        - name: Ensure validated-patterns-secrets namespace absent
+          ansible.builtin.include_role:
+            name: manage_secret_namespace
+          vars:
+            app_namespace: validated-patterns-secrets
+            namespace_state: absent
+
+    - name: Show git diff reminder
+      ansible.builtin.shell:
```

Comment:
We should definitely except this from the lint action. ("noqa" is the normal way to do this, on the offending line)